.github/workflows/publish.yml

```
@@ -0,0 +1,52 @@
+name: Publish packages
+
+on:
+  workflow_dispatch:
+    inputs:
+      sha:
+        description: 'Commit SHA to release from'
```

There is no need for quotes here, no?

```suggestion
        description: Commit SHA to release from
```

But it's probably a bigger problem, why is prettier not covering the yml files.